### PR TITLE
Overhaul documentation site look

### PR DESCRIPTION
**Pull request by alicerunsonfedora**
This PR makes the following pages:

- Move sections to top has horizontal tabs to better organize the site
- Move NotificationKit and ScreenKit documentation into Frameworks
- Rename AppKit documentation to Apps
- Change the banner color to gray
- Replace the generic icon with the AliceOS logo (A with halos)

image

### CODE REVIEW

**Comment by alicerunsonfedora** [comment]
Updated PR Changes:

- Renames "Apps" to "Develop Apps" and places AppKit documentation in Frameworks
- Renames "Getting Started" page to "Installing AliceOS" to prevent possible section confusion
- Adds "Recommended documentation" to applicable pages
- Cleans up documentation on developing apps

File: mkdocs/Develop-Apps/03-interface-design.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    Resources/
+    Views/
+        ScreenName.rpy
+        SecondScreen.rpy
+    Styles/
+        ScreenStyles.rpy
+    AppManifest.rpy
+```
+
+## Suggested Documentation
```
Comment (GanstaKingofSA) [inline mkdocs/Develop-Apps/03-interface-design.md]: The last 2 files have Suggested Documentation formatted as `Suggested documentation`. Minor spelling error here but noting here

File: mkdocs/Develop-Apps/04-watching-events.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+Runs any initial tasks or specified startup tasks during boot.
+
+### `applicationShouldLaunchAtLogin(self)`
+
+Determines whether the application has permission to run tasks during boot.
+
+**Returns**: Boolean value dictating if the app has permission to run at boot.
+
+## Suggested documentation
```
Comment (GanstaKingofSA) [inline mkdocs/Develop-Apps/04-watching-events.md]: Also applies

File: mkdocs/Develop-Apps/03-interface-design.md (hunk 1 above)
Comment (alicerunsonfedora) [reply mkdocs/Develop-Apps/03-interface-design.md]: Yeah, should be `Suggested documentation`. Should be fixed now

**Pull request merged**